src/GlobalStyles/global.style.ts

```diff
@@ -1,13 +1,15 @@
-import { createGlobalStyle } from "styled-components";
```

Comment:
It looks like I used squashed into the wrong direction, `global.style.ts `and `index.ts` shouldn't be behind that commit message.